content/docs/installation/supported-releases.md

```diff
@@ -10,31 +10,34 @@ Inspired by https://istio.io/latest/about/supported-releases/
 This page lists the status, timeline and policy for currently supported releases.
 
 Each release is supported for a period of four months, and we aim to create a new
-release every two months.
+release roughly every two months, accounting for holiday periods, major conferences
+and other world events.
 
 <h2 id="supported-releases">Currently supported releases</h2>
 
 | Release  | Release Date |   End of Life   | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |----------|:------------:|:---------------:|:----------------------------------:|:---------------------------------:|
-| [1.10][] | Oct 17, 2022 | Release of 1.12 |            1.20 → 1.25*            |            4.7 → 4.11             |
-| [1.9][]  | Jul 22, 2022 | Release of 1.11 |            1.20 → 1.24*            |            4.7 → 4.11             |
+| [1.10][] | Oct 17, 2022 | Release of 1.12 |            1.20 → 1.25\*            |            4.7 → 4.11             |
+| [1.9][]  | Jul 22, 2022 | Release of 1.11 |            1.20 → 1.24\*            |            4.7 → 4.11             |
```

Comment:
Note to self: look more closely at the OpenShift support, because RedHat themselves seem to have dropped support for 4.7 in their OperatorHub packaging infrastructure:
 * https://github.com/redhat-openshift-ecosystem/community-operators-prod/issues/1775